com.unity.render-pipelines.high-definition/Editor/Material/StackLit/ShaderGraph/StackLitPass.template

```diff
@@ -392,18 +392,9 @@ Pass
 
         // normal delivered to master node
```

Comment:
you missed to remove $NormalDropOffTS:		                            #define _NORMAL_DROPOFF_TS 1 in stacklit

Reply:
Do a pass on all _NORMAL_DROPOFF_TS / _NORMAL_DROPOFF_OS / _NORMAL_DROPOFF_WS in the code 
and NormalDropOffTS  / NormalDropOffOS / NormalDropOffWS

in C# and hlsl / template .shader

Reply:
oh universal use it, so only remove the thing that are on HDRP side